src/neo-vm/Types/Map.cs

```diff
@@ -30,8 +31,9 @@ public class Map : CompoundType, IReadOnlyDictionary<PrimitiveType, StackItem>
         }
 
         public override int Count => dictionary.Count;
-        public override int ItemsCount => dictionary.Count * 2;
         public IEnumerable<PrimitiveType> Keys => dictionary.Keys;
+        internal override IEnumerable<StackItem> SubItems => Keys.Concat(Values);
```

Comment:
This is the final fix! :) thanks for it :D .... but if you want to make me happy. I don't like to see `Linq` inside the VM hahah.

```suggestion
        internal override IEnumerable<StackItem> SubItems()
        {
                foreach (var key in Keys) yield return key;
                foreach (var value in Values) yield return value;
        }
```

If you don't like it, I will try to convince you in the future :P

Reply:
In fact they are exactly the same.

Reply:
This is how linq is implemented internally.